go/pkg/db/identity_insert_ratelimit.sql_generated.go

```diff
@@ -29,7 +29,12 @@ INSERT INTO ` + "`" + `ratelimits` + "`" + ` (
     ?,
     ?,
     ?
-)
+) ON DUPLICATE KEY UPDATE
```

Comment:
this just fixes the 500 error I had last night, right? and has nothing to do with bulks, or am I missing something?

Reply:
ah no.. I had that 500 when creating an override
so why is this here?

Reply:
The old bulk insert function had a custom statement with the on duplicate key but the normal one didnt because we just upsert everything